x-pack/legacy/plugins/siem/public/containers/source/index.tsx

```diff
@@ -57,30 +57,28 @@ interface WithSourceProps {
   sourceId: string;
 }
 
-export const WithSource = React.memo<WithSourceProps>(({ children, sourceId }) => {
-  const getIndexFields = (title: string, fields: IndexField[]): StaticIndexPattern =>
+const getIndexFields = memoizeOne(
```

Comment:
As with the other memoize-one comment, may want a short comment here to clarify the change to `memoizeOne` over `React.memo`?

>So, memoize-one (in common with most memoization libraries) remembers the result of a given function with a set of arguments, despite where the last execution occurs. React.memo, on the other hand, is for memoizing a single occurrence of a component when attempting to re-render, and it will not work outside of its instance.

From: https://medium.com/better-programming/react-memo-vs-memoize-71f85eb4e1a#eb1f